test/functional/page_objects/settings_page.js

```diff
@@ -281,7 +281,8 @@ export function SettingsPageProvider({ getService, getPageObjects }) {
     }
 
     async clickIndexPatternLogstash() {
-      await find.clickByPartialLinkText('logstash-*');
+      const indexLink = await find.byXPath(`//a[descendant::*[text()='logstash-*']]`);
+      await indexLink.click();
```

Comment:
`clickByPartialLinkText` is selecting first link that **contains** `logstash-*`, e.g. `long-window-logstash-* `which cause some tests to fail in Firefox because of sorting bug #35551

Using xpath we strictly search for element with `logstash-*` text

Reply:
Eeek, XPath! Your logic is sound as to why the existing option doesn't work, but maybe we can use `find.clickByLinkText()` instead?

Reply:
I know we don't want to use XPath, but unfortunately `find.clickByLinkText('logstash-*');` doesn't work:
```
         └- ✖ fail: "management  scripted fields should not allow saving of invalid scripts"
         │      retry.try timeout: TimeoutError: Waiting for element to be located By(link text, logstash-*)
         │ Wait timed out after 10023ms
``` 

I don't have any alternative to XPath
<img width="1502" alt="Kibana 2019-05-02 13-23-04" src="https://user-images.githubusercontent.com/10977896/57072472-173e2900-6cde-11e9-98c4-d36c192a3023.png">